### PR TITLE
Discussion on the packaging of Python code

### PR DESCRIPTION
**Pull request by robinandeer**
One conventions that is quite common in Python is to avoid using `_` (underscore) in module names. It's fine for variables etc. but when it's packages and modules I try to stick to single words or just concatenate them without separator: `count_variants` -> `countvariants` or perhaps something like `vcfcount` 🙂

### CODE REVIEW

**Comment by robinandeer** [comment]
I like to use `pip install --editable .` in favour of `python setup.py develop` -- then it's always about `pip` when you are installing something, or what do you think?

**Comment by robinandeer** [comment]
Really nice overview and tutorial on how to setup a new Python project 👍🎉

**Comment by moonso** [comment]
Sure thats fine. I'm not quite certain what anyone of those commands do anyway 😆

**Comment by moonso** [comment]
Thx

File: python/count_variants/count_module.md [hunk 1]
```diff
@@ -5,12 +5,13 @@ When we finnish this section we should have a small program that counts the numb
 We will try to keep the code separated inside our projekt as described [here](../conventions.md), we store the counting code in a submodule called utils and the cli code in a module called cli. So run:
 
 ```bash
-$ mkdir count_variants/cli count_variants/cli
```
Comment (robinandeer) [inline python/count_variants/count_module.md]: my editor informed me that it's best practise to exclude the dollar sign unless you are showing example output

File: python/count_variants/installing.md [hunk 2]
```diff
@@ -18,11 +19,12 @@ We are telling `setup.py` that if someone write `count_variants` on the command
 
 Now we are going to run the command that installs all dependencies and the package itself:
 
+```bash
+pip install --editable .
```
Comment (robinandeer) [inline python/count_variants/installing.md]: see comment ...

File: python/count_variants/structure.md [hunk 3]
```diff
@@ -26,34 +27,37 @@ Every decent python package should at least include a README file that describes
...
 
 open the file with an [editor](../../editors/README.md) of your choice and write the following:
 
-```
+```markdown
 # count_variants
 A package to count variants in VCF files based on different criterias.
 ```
 
+> You might recognize this as "markdown", a markup language for formatting text. You can read more about it under [Documentation](../../documentation/README.md).
```
Comment (robinandeer) [inline python/count_variants/structure.md]: just added a little hint on where to find more info about markdown

File: python/count_variants/structure.md [hunk 4]
```diff
@@ -72,19 +76,13 @@ Untracked files:
 	requirements.txt
 
 nothing added to commit but untracked files present (use "git add" to track)
-$ git add .gitignore
-$ git add README.md
-$ git add LICENSE
-$ git add MANIFEST.in
-$ git add requirements.txt
+$ git add .gitignore README.md LICENSE MANIFEST.in requirements.txt
```
Comment (robinandeer) [inline python/count_variants/structure.md]: added all files in one line

Comment (moonso) [reply python/count_variants/structure.md]: Ok can we merge this one? I'm planning to add something about testing

**Comment by moonso** [comment]
Ok can we merge this one? I'm planning to add something about testing

**Comment by robinandeer** [comment]
I take it as you signing off about the updates then!

**Comment by moonso** [comment]
Oh yes

**Pull request merged**